nni/algorithms/compression/v2/pytorch/pruning/tools/task_generator.py

```diff
@@ -187,6 +187,11 @@ def __init__(self, origin_model: Module, origin_config_list: List[Dict], origin_
     def reset(self, model: Module, config_list: List[Dict] = [], masks: Dict[str, Dict[str, Tensor]] = {}):
         self.current_temperature = self.start_temperature
 
+        # TODO: replace with validation here
+        for config in config_list:
+            if 'sparsity' in config or 'sparsity_per_layer' in config:
+                _logger.warning('Only `total_sparsity` can be differentially allocate sparse ratio to each layer, `sparsity` or `sparsity_per_layer` will allocate fix sparse ratio to layers. Make sure you know what this will lead to, otherwise please use `total_sparsity`.')
```

Comment:
There is a grammar problem, please fix it.

Reply:
`allocate` -> `allocated`, `fix` -> `fixed`